### PR TITLE
[MODORDERS-902]-Added Karate test.

### PR DESCRIPTION
**Pull request by SinghAdes**
## Purpose
_Describe the purpose of the pull request._
https://issues.folio.org/browse/MODORDERS-902
## Approach
_How does this change fulfill the purpose?_

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/find-holdings-by-location-and-instance-for-mixed-pol-with-create-inventory-none.feature [hunk 1]
```diff
@@ -0,0 +1,166 @@
+@parallel=false
+# for https://issues.folio.org/browse/MODORDERS-871
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/find-holdings-by-location-and-instance-for-mixed-pol-with-create-inventory-none.feature:2]: incorrect story number

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/find-holdings-by-location-and-instance-for-mixed-pol-with-create-inventory-none.feature [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    * def orderId2 = callonce uuid4
+    * def poLineId1 = callonce uuid5
+    * def poLineId2 = callonce uuid6
+
+  Scenario: Create finances
+    * configure headers = headersAdmin
+    * call createFund { 'id': '#(fundId)'}
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 10000, 'fundId': '#(fundId)'}
+
+  Scenario: Create first order
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/find-holdings-by-location-and-instance-for-mixed-pol-with-create-inventory-none.feature:30]: For simple scanarious you can use already written functions to create/open orders like this as example:

Scenario: Create composite order
    * def v = call createOrder { id: #(orderId) }

  Scenario: Create order line
    * def v = callonce createOrderLine { id: #(poLineId), orderId: #(orderId), fundId: #(globalFundId) }

  Scenario: Open order
    * def v = callonce openOrder { orderId: "#(orderId)" }
    
These functions located in reusable package: https://github.com/folio-org/folio-integration-tests/tree/master/acquisitions/src/main/resources/thunderjet/mod-orders/reusable

Comment (SinghAdes) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/find-holdings-by-location-and-instance-for-mixed-pol-with-create-inventory-none.feature:30]: @SerhiiNosko :

Yes , i have seen those, create orderLine is  reading different file and particulars are also different , thats why i written. 
if i  want to reuse, i need to write seperate feature file. 

let me know if  i can do this . thanks

Comment (SerhiiNosko) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/find-holdings-by-location-and-instance-for-mixed-pol-with-create-inventory-none.feature:30]: I see, okay lets use as is, no changes required

Comment (SinghAdes) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/find-holdings-by-location-and-instance-for-mixed-pol-with-create-inventory-none.feature:33]: fixed.

**Review by Dmitriy-Butramyou — APPROVED** [review]

**Pull request merged**